src/translations/en.json

```diff
@@ -3178,7 +3178,7 @@
           "help": "Help",
           "start_conversation": "Start conversation",
           "reload_resources": "Reload resources",
-          "exit_edit_mode": "Exit UI edit mode",
+          "exit_edit_mode": "Done",
```

Comment:
Should I create a new translation? Other languages are going to be a longer title for this

Reply:
It's fine.